site/en/docs/extensions/mv3/xhr/index.md

```diff
@@ -214,6 +182,11 @@ If you modify the default [Content Security Policy][12] for your extension by ad
 you'd like to connect are allowed. While the default policy doesn't restrict connections to hosts,
 be careful when explicitly adding either the `connect-src` or `default-src` directives.
 
+### Using  Fetch() vs. XMLHttpRequest()
+
+`fetch()` was created specifically for service workers and follows a broader web trend away from synchronous operations. The `XMLHttpRequest()` API is supported, but calling `XMLHttpRequest()` will trigger the service worker's fetch handler. New work should favor the use of `fetch()` wherever possible.
```

Comment:
Tweaks this a bit to highlight:

- `XMLHttpRequest` is not available in service workers
- I removed the brackets from `XMLHttpRequest`. I know we usually do that for functions, but I think this is an exception - it's a constructor so `XMLHttpRequest()` without `new XMLHttpRequest()` is invalid syntax and I think the brackets make things more confusing.

```suggestion
`fetch()` was created specifically for service workers and follows a broader web trend away from synchronous operations. The `XMLHttpRequest` API is supported in extensions outside of the service worker, and calling `XMLHttpRequest` will trigger the service worker's fetch handler. New work should favor the use of `fetch()` wherever possible.
```